tests/test_api.py

```diff
@@ -103,7 +103,150 @@ def test_bad_id(client):
         response.raise_for_status()
 
     # TODO: should move to a different func
+    # Testing invalid box bounds
     response = client.post(
         "api/v1/source/box",
         json={"ra_min": 1, "ra_max": 0, "dec_min": 1, "dec_max": 0},
     )
+
+
+###################################################
+#              Astroquery Tests                   #
+###################################################
+
+
+def test_add_and_remove_service(client):
+    response = client.put(
+        "api/v1/service/new", json={"name": "Simbad", "config": "test"}
```

Comment:
If you want to split things up, it's usually better to just use different files than do this.